api/src/main/java/com/velocitypowered/api/proxy/Player.java

```diff
@@ -45,6 +46,21 @@
    */
   String getUsername();
 
+  /**
+   * The locale the velocity proxy is translating its messages to. By default {@link PlayerSettings#getLocale()}
```

Comment:
Still not satisfied with this Javadoc. The description is very imprecise - we are not necessarily _translating_ messages at this level.

Reply:
Is it better now?

Reply:
No, it's not. Perhaps `Returns the locale the proxy will use to send messages translated via the Adventure global translator. By default, the value of {@link PlayerSettings#getLocale()} is used.`.

Reply:
Is it good now?